recipe/scripts/deactivate.sh

```diff
@@ -5,6 +5,6 @@ unset PROJ_LIB
 unset PROJ_NETWORK
 
 if [ -n "$_CONDA_SET_PROJ_LIB" ]; then
-    export PROJ_LIB=$_CONDA_SET_PROJ_LIB
+    export PROJ_LIB="${_CONDA_SET_PROJ_LIB:-}"
```

Comment:
I don't think this is the line that needs fixing.  It's the one above.
```suggestion
    export PROJ_LIB=$_CONDA_SET_PROJ_LIB
```